gandalf.js

```diff
@@ -1,89 +1,95 @@
-/**
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?